tests/dragonfly/replication_test.py

```diff
@@ -332,6 +334,51 @@ async def start_master():
         await wait_available_async(c_replica)
         assert await seeder.compare(capture, port=replica.port)
 
+"""
+Test re-connecting replica to different masters.
```

Comment:
did we have bug in this scenario?

Reply:
No we didn't, but we tested only re-connecting to the same master. Either way its costs us only 10 seconds or so